BUILDING.md

```diff
@@ -385,3 +385,21 @@ and [user guide](https://openssl.org/docs/fips/UserGuide-2.0.pdf).
    `/usr/local/ssl/fips-2.0`
 8. Build Node.js with `make -j`
 9. Verify with `node -p "process.versions.openssl"` (for example `1.0.2a-fips`)
+
+## Building Node.js with external core modules
+
+In general those core modules are only located in `lib/` folder.  Node.js can
+specify one or more JavaScript text files to be bundled in the binary as
+builtin modules.
+
+##### Unix / macOS
+
+```console
+$ ./configure --link-module '/root/myModule.js'
```

Comment:
Can you add an explanation for this command? Maybe something like <code>To make \`/root/myModule.js\` available via \`require('root/myModule')\`:</code>